features/step_definitions/cookies_steps.rb

```diff
@@ -0,0 +1,24 @@
+When('I see the cookies banner') do
+  expect(page).to have_css '.govuk-cookie-banner'
+  expect(page.find('.govuk-cookie-banner').visible?).to eq true
+  expect(page).to have_content I18n.t('layouts.cookie_banner.accept_button')
+end
+
+Then('I see the {string} confirmation message') do |string|
+  expect(find('.govuk-cookie-banner__confirmation').visible?).to eq true
+  expect(find('.govuk-cookie-banner__confirmation')).to have_content 'Your cookie settings were saved'
+end
+
+Then('the cookie banner is not available') do
+  expect(page).not_to have_selector('.govuk-cookie-banner')
+end
+
+When('I choose to turn cookies {string}') do |status|
+  option_label = "cookies.new.analytics_cookie_#{status}_label"
+  choose(I18n.t(option_label))
+end
+
+Then('the cookie preference is saved') do
+  expect(find('.govuk-notification-banner--success').visible?).to eq true
+  expect(page).to have_content I18n.t('cookies.new.cookie_notification')
+end
```

Comment:
you can use site prism to define a cookie banner section and then use syntax like
```
  expect(cooke_banner).to be_visible
  expect(cooke_banner).to have_content "expected_text"
```

see similar setups for other pages

Reply:
I could only make it work by creating a `CookiePage` and including the section rather than setting a standalone `CookieSection`, even though the documentation seems to show it is possible. It seems counterintuitive though since the cookie banner is just a section?

When I try `CookieSection.new`, I get `wrong number of arguments (given 0, expected 2) (ArgumentError)`.

Reply:
Perhaps it'd make more sense for me to add the element to `ExternalUserHomePage` or `UserHomePage`? @jsugarman

Reply:
Its not a blocker anyway, so if you want to do as a refactor after the fact that is fine. There is a hierarchy with `BasePage` as the superclass. You could put it somewhere in that hierarchy that makes sense for the cookie banner "section"